vector/src/main/java/im/vector/app/features/createdirect/CreateDirectRoomViewModel.kt

```diff
@@ -71,10 +71,13 @@ class CreateDirectRoomViewModel @AssistedInject constructor(@Assisted
      * If users already have a DM room then navigate to it instead of creating a new room.
      */
     private fun onSubmitInvitees(action: CreateDirectRoomAction.CreateRoomAndInviteSelectedUsers) {
-        if (action.existingDmRoomId != null) {
+        val existingRoomId = action.selections.singleOrNull()?.getMxId()?.let { userId ->
```

Comment:
This does not look right. If it's a request for a grouped DM this will just open the DM with the first user?

Reply:
singleOrNull() ≠ firstOrNull()

Reply:
Indeed, agree to check for existing grouped DM, requires some more changes